js/main.js

```diff
@@ -0,0 +1,153 @@
+    });
+  });
+};
+
+//---------------------- Delete from cart --------------------------------
+const deleteEventBtn = () => {
+  const btnDelete = document.querySelectorAll('.delete-btn');
+
+  btnDelete.forEach(button => {
+    button.addEventListener('click', () => {
+      const productTitle = button.getAttribute('data-title');
+      const productId = button.getAttribute('data-id');
+      const cartList = JSON.parse(localStorage.getItem('cart'));  
+
+      console.log(productId);
+      console.log(productTitle);
+      
+      const updatedCartList = cartList.filter(item => item.id !== +productId);
+      console.log(updatedCartList);
+      localStorage.setItem('cart', JSON.stringify(updatedCartList));
+      location.reload();
+    })
+  })
+};
+
+//---------------------- Process events --------------------------------
+const addProcessEvent = () => {
+  const prevBtns = document.querySelectorAll('.prev-btn');
+  const nextBtns = document.querySelectorAll('.next-btn');
+  const progressLine = document.getElementById('progress')
+  const processBlocks = document.querySelectorAll('.process-block');
+  const progressSteps = document.querySelectorAll('.progress-step');
+
+  let processStepsNum = 0;
+
+  nextBtns.forEach(button => {
+    button.addEventListener('click', (e) => {
+      e.preventDefault();
+
+      processStepsNum++;
+      updateProcessBlocks();
+      updateProgressBar();
+    })
+  });
+
+  prevBtns.forEach(button => {
+    button.addEventListener('click', (e) => {
+      e.preventDefault();
+
+      processStepsNum--;
+      updateProcessBlocks();
+      updateProgressBar();
+    })
+  });
+
+  const updateProcessBlocks = () => {
+    processBlocks.forEach(processBlock => {
+      if(processBlock.classList.contains('process-block-active')) {
+        processBlock.classList.remove('process-block-active');
+      }
+    });
+
+    processBlocks[processStepsNum].classList.add('process-block-active')
+  };
+
+  const updateProgressBar = () => {
+    progressSteps.forEach((step, index) => {
+      if(index <= processStepsNum) {
+        step.classList.add('progress-step-active');
+      } else {
+        step.classList.remove('progress-step-active');
+      }
+    });
+
+    const progressActive = document.querySelectorAll('.progress-step-active');
+
+    progressLine.style.width = ((progressActive.length - 1) / (progressSteps.length - 1)) * 100 + "%";
+  };
+};
+
+const addSubmitEvent = () => {
+  const submitBtn = document.querySelector('.submit-btn');
+
+  if(submitBtn) {
+    submitBtn.addEventListener('click', (e) => {
+      e.preventDefault();
+  
+      const name = document.getElementById('name').value;
+      const phoneNumber = document.getElementById('phone-number').value;
+      const email = document.getElementById('email').value;
+      const address = document.getElementById('address').value;
+      const customer = {name, phoneNumber, email, address};
+  
+      fetch('http://localhost:3000/customer', {
```

Comment:
Những thứ như URL thế này cho vào file constant
Với tên biến toàn bộ in hoa. Nhưng sao lại là customer nhỉ :v
VD: USER_API_URL

Reply:
Customer là khách hàng mà e. A tưởng phần đó là thông tin khách hàng chứ nhỉ